subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/tasks/ReleaseNeeded.java

```diff
@@ -4,6 +4,7 @@
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.Logging;
 import org.shipkit.gradle.release.ReleaseNeededTask;
+import org.shipkit.internal.util.ArgumentValidation;
 import org.shipkit.internal.util.EnvVariables;
 
 public class ReleaseNeeded {
```

Comment:
Can you document the ReleaseNeededPlugin better in the Javadoc? I would love to see the documentation how to use it. There is also "docs/gradle-plugins/release-needed-plugin.md" docs that we use for this feature.

Let's build great docs!!!

Reply:
Sure, I will improve the (java)doc later on today.